includes/blocks/class-sensei-course-blocks.php

```diff
@@ -86,7 +118,7 @@ public function enqueue_block_editor_assets() {
 			return;
 		}
 
-		Sensei()->assets->enqueue( 'sensei-blocks', 'blocks/index.js', [], true );
+		Sensei()->assets->enqueue( 'sensei-blocks', 'blocks/course.js', [], true );
```

Comment:
The name could also be updated here, for something like `sensei-single-course-blocks`.

Reply:
Done in 2dfdd8595c65724589ceeb63ff77231b577fd1b9